### DISCUSSION TITLE
Can't find the enriched motif in cisbp database

### DISCUSSION THREAD

**Discussion by LI-Yan-Ming** [GreenleafLab/ArchR#1781] [Questions / Documentation]
Hi,
Thank you for developing such a good tool for scATAC-seq analysis!
Following the tutorial, I have identified the enriched motifs from the differential peaks. One enriched motif from my data is Smarcc1 (motifSet = "cisbp"). I tried to get the motif sequence of Smarcc1 from the CIS-BP database (http://cisbp.ccbr.utoronto.ca/). However, I can not find this transcription factor in cisbp database.
I am wondering where can I get the motif sequence of the transcription factors that been analyzed in ArchR. 
Thank you so much! Any comments will be much appreciated!

**Comment by rcorces** [comment]
ArchR uses motif databases curated by the ~~motifmatchr~~ chromVAR package. If you dont find these sufficient, you can make your own custom PWMatrixList.

**Reply by LI-Yan-Ming** [reply]
Thank you for the explanation! 
I guess I have a better understanding of how this works now. motifmatchr was used for finding motifs that are present in peaks. In my case, motif set to be used for annotation is "cisbp" motif sets from the chromVAR package. I can not find my Smarcc1 in CIS-BP database because they have released a new version, but Smarcc1 is only presented in the previous version. 
Just type this down in case anyone else has a similar problem as mine.
Thanks again!

**Reply by rcorces** [reply]
yes sorry in my haste I type "motifmatchr" above but I meant chromVAR as you said. I've edited that post.
